HealthyTests/Classes/Modules/Splash/SplashViewControllerTest.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+@testable import Healthy
+
+final class SplashViewControllerTest: XCTestCase {
+
+    // MARK: Properties
+
+    private var viewController: SplashViewController!
+    private var viewModelMock: SplashViewModelSpy!
+
+    // MARK: Lifecycle
+
+    override func setUpWithError() throws {
+        super.setUp()
+        viewModelMock = SplashViewModelSpy()
+        viewController = SplashViewController(viewModel: viewModelMock)
+        viewController.loadViewIfNeeded()
+    }
+
+    // MARK: Tests
+
+    func test_didTapStartCooking_shouldCallViewModelSplash() {
+        // When
+        viewController.didTapStartCooking(UIButton())
+
+        // Then
+        XCTAssertEqual(viewModelMock.performStartCookingCallCount, 1)
+    }
+}
+
+// MARK: SplashViewModelSpy
+
+private final class SplashViewModelSpy: SplashViewModelType {
```

Comment:
```suggestion
private final class SplashViewModelMock: SplashViewModelType {
```

Reply:
I made it SplashViewModelMock and I am not the one who changed it to SplashViewModelSpy